project/plugins.sbt

```diff
@@ -1,10 +1,10 @@
 resolvers in Global += Resolver.sonatypeRepo("public")
 
 addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.5.1")
-//addSbtPlugin("ch.epfl.scala"      % "sbt-scalajs-bundler"      % "0.20.0")
 addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.4.2")
 addSbtPlugin("edu.gemini"         % "sbt-lucuma"               % "0.3.7")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
 addSbtPlugin("com.eed3si9n"       % "sbt-buildinfo"            % "0.10.0")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-git"                  % "1.0.0")
 addSbtPlugin("ch.epfl.scala"      % "sbt-scalafix"             % "0.9.27")
+addSbtPlugin("com.timushev.sbt"   % "sbt-rewarn"               % "0.1.3")
```

Comment:
This will continue to emit all existing warnings on every compile, even if the source where the warns are is not recompiled. Makes sure we don't push to CI with warnings.

Reply:
Cool